### PR TITLE
Add StakedICP showcase

### PR DESCRIPTION
**Pull request by 0xAegir**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Review by sesi200 — APPROVED** [review]

**Comment by 0xAegir** [comment]
Ok, resolved the conflicts. Should be good-to-go.

File: showcase.json [hunk 1]
```diff
@@ -1942,5 +1942,21 @@
...
+    ],
+    "description": "StakedICP is the non-custodial liquid-staking protocol for ICP. ICP is staked in the NNS DAO, and stakers receive rewards daily. The stICP token is DeFi-compatible, to support protocols building on the Internet Computer, and always fully-backed by ICP staked in the NNS.",
+    "usesInternetIdentity": true,
+    "website": "https://stakedicp.com",
+    "github": "https://github.com/AegirFinance/StakedICP",
+    "twitter": "https://twitter.com/StakedICP",
+    "display": "Normal",
+    "logo": "/img/showcase/stakedicp_logo.webp",
+    "screenshots": []
+  },
```
Comment (sesi200) [inline showcase.json:1961]:
```suggestion
  }
```

Comment (sesi200) [reply showcase.json:1961]: the JSON breaks with the extra comma

**Review by sesi200 — APPROVED** [review]

**Comment by domwoe** [comment]
@0xAegir Would you mind resolving the conflicts?

**Comment by 0xAegir** [comment]
Alright, resolved again. 🤞

**Review by domwoe — APPROVED** [review]

**Pull request merged**